docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -97,6 +97,43 @@ supports the following features:
   :ref:`iceberg-materialized-views`
 * :ref:`sql-views-management`
 
+.. _iceberg-alter-table-execute:
+
+ALTER TABLE EXECUTE
```

Comment:
Change this section to be about ALTER TABLE EXECUTE OPTIMIZE .. since thats all it is ..